### DISCUSSION TITLE
Docs say: "Or use any other front-end framework if you prefer."

### DISCUSSION THREAD

**Discussion by cameronelliott** [thuliteio/doks#152] [Q&A] [answered]
@h-enk 
Henk,
I really have fallen in love with Doks, and I am kind of figuring out how to best use it.

When reading the documentation, I came across this:
> Build robust, flexible, and intuitive websites with Bootstrap. Or **_use any other front-end framework if you prefer_**.

From this page:
https://getdoks.org/docs/reference-guides/bootstrap/

But, I must admit, this is a bit confusing for me, aren't Doks and Bootstrap very tightly coupled?
I know Bulma much better than Bootstrap, but isn't it impractical for me to use Bulma with Doks?
I suppose I could mix the two frameworks somehow, but that seems like a bad idea.

Would you please explain this statement in the documentation so I can understand your meaning?

Thanks,
Cameron

**Comment by h-enk** [comment, marked as answer]
Hey @cameronelliott, thanks — great to hear!

> [..] aren't Doks and Bootstrap very tightly coupled?

You're absolutely right. Would be a major effort to translate Doks to Bulma (or any other framework).

I will remove the phrase "Or use any other front-end framework if you prefer." throughout the Doks site (it was copied over from Hyas — for which it is valid).

Hope you're not having that much of a hard time with Bootstrap (I wouldn't expect so).

**Reply by siduck** [reply]
@h-enk I would like to use a web framework such as preact or solidjs over plain html, how do i that :thinking:

**Reply by h-enk** [reply]
That would be out of scope for Doks (Discussions). Aside from that, I wouldn't know/ haven't done that before. But, since Doks uses npm as a dependency manager, I would start by adding the necessary dependencies. For instance for SolidJS: `npm install solid-js babel-preset-solid` (but I guess you got that far already).
